mozci/console/commands/push.py

```diff
@@ -56,61 +94,66 @@ def handle(self):
                 "<comment>Provided --output pointed to a inexistent directory that is now created.</comment>"
             )
 
-        push = Push(self.argument("rev"), self.argument("branch"))
-        classification, regressions = push.classify(
-            confidence_medium=medium_conf, confidence_high=high_conf
-        )
-        self.line(
-            f'Push associated with the head revision {self.argument("rev")} on the branch '
-            f'{self.argument("branch")} is classified as {classification.name}'
-        )
-
-        if output:
-            to_save = {
-                "push": {
-                    "id": push.push_uuid,
-                    "classification": classification.name,
-                },
-                "failures": {
-                    "real": {
-                        group: [
-                            {"task_id": task.id, "label": task.label}
-                            for task in failing_tasks
-                        ]
-                        for group, failing_tasks in regressions.real.items()
+        for push in pushes:
```

Comment:
I noticed `cleo` has a progress feature that could be useful here:
https://cleo.readthedocs.io/en/latest/helpers/progress_bar.html

Though I guess if there is a ton of output being emitted from `mozci` it won't work. We'd need to only show the progress bar if other output is suppressed.. probably not worth implementing.

Reply:
I think that would be a nice followup, but it mayneed to be integrated a bit more deeply into mozci

Reply:
Follow-up filed: https://github.com/mozilla/mozci/issues/547